app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -1,12 +1,20 @@
 import * as React from 'react'
 import { Repository } from '../../models/repository'
 import { Octicon, iconForRepository } from '../octicons'
-import { CloningRepository } from '../../lib/dispatcher'
-import { showContextualMenu } from '../main-process-proxy'
+import { showContextualMenu, IMenuItem } from '../main-process-proxy'
+import { Repositoryish } from './group-repositories'
```

Comment:
I know this is not really a part of this PR, but can we come up with a better name for `Repositoryish`? IMO, it makes more sense to call rename `Repository` -> `LocalRepository` and `Repositoryish` -> `Repository`.